src/printer.js

```diff
@@ -499,11 +499,20 @@ function genericPrint(path, options, print) {
       if (node.arguments.length === 0) {
         return node.functionName;
       }
-      // @TODO: add call args
```

Comment:
@mattiaerre I deleted this comment because it seems outdated, but let me know if there's something pending here.